tasks.c

```diff
@@ -719,6 +719,32 @@ static void prvAddNewTaskToReadyList( TCB_t * pxNewTCB ) PRIVILEGED_FUNCTION;
     extern void vApplicationMinimalIdleHook( void );
 #endif /* #if ( configUSE_MINIMAL_IDLE_HOOK == 1 ) */
 
+#if ( ( configUSE_TRACE_FACILITY == 1 ) && ( configUSE_STATS_FORMATTING_FUNCTIONS > 0 ) )
+
+/*
+ * Convert the snprintf return value to the number of characters
+ * written. The following are the possible cases:
+ *
+ * 1. The buffer supplied to snprintf is large enough to hold the
+ *    generated string. The return value in this case is the number
+ *    of characters actually written, not counting the terminating
+ *    null character.
+ * 2. The buffer supplied to snprintf is NOT large enough to hold
+ *    the generated string. The return value in this case is the
+ *    number of characters that would have been written if n had
+ *    been sufficiently large, not counting the terminating null
+ *    character.
+ * 3. Encoding error. The return value in this case is a negative
+ *    number.
+ *
+ * From 1 and 2 above ==> Only when the return value is non-negative
+ * and less than the supplied buffer length, the string has been
+ * completely written.
+ */
+    static size_t prvSnprintfReturnValueToCharsWritten( int lSnprintfReturnValue,
```

Comment:
```int lSnprintfReturnValue;```
`int` is not list in [FreeRTOS naming convention](https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html). However, maybe we can use `int iSnprintfReturnValue` to be consistent with `int iTaskErrno`. The suggestion also applied to the following int variables.

Reply:
Updated.